modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,146 @@
+
+See xref:security/authentication.adoc[] for more information on creating and managing superusers.
+
+When performing an upgrade from older versions of Redpanda, all existing SASL/SCRAM users are assigned to the `Users` role. This role has no assigned ACLs. You will need to use your `superuser` account to create additional roles and assign appropriate ACLs as necessary.
+
+==== Policy conflicts
+
+You can assign a combination of ACLs and roles to any given principal. ACLs allow permissions, deny permissions, or specify a combination of both. As a result, users may at times have role assignments with conflicting policies. If this situation arises, the user is permitted to perform an operation if and only if:
+
+* There does not exist a DENY permission matching the operation.
+* There exists an ALLOW permission matching the operation.
+
+=== RBAC Example
+
+Consider a scenario where you have a set of private topics used by your software engineers to publish application update information to users. All your private topics begin with the prefix `private-`. You might create a new `Software Eng.` role to represent the software engineers given permission to write to these private topics. You would then assign the `Software Eng.` role as the `allowed-role` for a new ACL specifying read and write permissions to `private-*`. Using a wildcard includes all existing private topics as well as any new private topics you might create later. You then assign the new role to Mike and Oren, your two software engineers who will write messages to this topic. The RPK commands to accomplish this are:
+
+[,bash]
+----
+rpk security role create 'Software Eng.'
+rpk acl create --operation read --operation write --topic private-* --allow-role 'Software Eng.'
+rpk security role assign 'Software Eng.' --principal User:mike
+rpk security role assign 'Software Eng.' --principal User:oren
+----
+
+This diagram shows the relationships between users, roles, and ACLs, including the example `Software Eng.` role:
+
+image::shared:rbac-overview.png[RBAC multiplicity]
+
+Now consider the situation where you want to create a new topic, `private-software-versions`, where users self-report the version of a component they are using. When using the ACL authorization mechanism, you would need to assign this ACL to every user in your Redpanda installation. Using RBAC allows you to make one update and apply it to everyone with that role. Adding the write permissions for this topic to the `User` role means everyone with that role (which is all users in our example) gains the access immediately. The rpk command to accomplish this is:
+
+[,bash]
+----
+rpk acl create --operation write --topic private-software-versions --allow-role User
+----
```

Comment:
I'm confused with:

>Adding the write permissions for this topic to the `User` role means everyone with that role (which is all users in our example) gains the access immediately.

In the example above, the role was named `'Software Eng.'`, but here you mention the role `User`.

Reply:
Yep! I'll update this to refer to the image and say something along the lines of, "as an extension of our example."

Reply:
@r-vasquez please review the language here now. I've updated it for clarity.